tests/media/server/ipc/sessionManagementServer/SessionManagementServerTestsFixture.cpp

```diff
@@ -182,12 +194,107 @@ void SessionManagementServerTests::serverWillSetLogLevels()
 
 void SessionManagementServerTests::sendServerInitialize()
 {
-    EXPECT_TRUE(m_sut->initialize(socketName, socketPermissions));
+    EXPECT_CALL(*m_linuxWrapperMock, chmod(testing::StrEq(socketName), socketPermissions)).WillOnce(Return(0));
```

Comment:
This EXPECT_CALL should be in serverWillInitialize.

We usually separate expect calls and invocation.